openy.install

```diff
@@ -324,3 +324,25 @@ function openy_update_8024() {
   // Enable module.
   \Drupal::service('module_installer')->install(['admin_toolbar']);
 }
+
+/**
+ * Enable Colorbutton module.
+ */
+function openy_update_8025() {
```

Comment:
This is not correct way of adding upgrade path.

1. Those changes should be added to https://github.com/ymcatwincities/openy/blob/8.x-1.x/config/install/editor.editor.full_html.yml
2. In order to see conflicts in upgrade path dashboard we should use `openy_upgrade_tool.param_updater` in order to see conflicts in the upgrade path dashboard.